evalml/tests/automl_tests/test_automl.py

```diff
@@ -5511,3 +5515,69 @@ def test_automl_passes_known_in_advance_pipeline_parameters_to_all_pipelines(
         ]
         == ["features", "date"]
     ).all()
+
+
+@pytest.mark.parametrize(
+    "data_splitter,mean_cv_is_none",
+    [(TrainingValidationSplit, True), (StratifiedKFold, False)],
+)
+def test_cv_validation_scores(
+    data_splitter,
+    mean_cv_is_none,
+    X_y_binary,
+    dummy_classifier_estimator_class,
+    dummy_binary_pipeline_class,
+    AutoMLTestEnv,
+):
+    X, y = X_y_binary
+    data_splitter = data_splitter()
+    automl = AutoMLSearch(
+        X_train=X,
+        y_train=y,
+        problem_type="binary",
+        max_batches=3,
+        data_splitter=data_splitter,
+        allowed_component_graphs={"Name": [dummy_classifier_estimator_class]},
+        n_jobs=1,
+    )
+    env = AutoMLTestEnv("binary")
+    with env.test_context(score_return_value={"Log Loss Binary": 0.5}):
+        automl.search()
+    cv_vals = list(set(automl.full_rankings["mean_cv_score"].values))
+    validation_vals = list(set(automl.full_rankings["validation_score"].values))
+    assert len(validation_vals) == 1
+    assert validation_vals[0] == 0.5
+    if mean_cv_is_none:
+        assert np.isnan(cv_vals[0])
+    else:
+        assert cv_vals[0] == validation_vals[0]
+
+
+def test_cv_validation_scores_time_series(
+    ts_data_binary,
+    AutoMLTestEnv,
+):
+    X, y = ts_data_binary
+    problem_configuration = {
+        "time_index": "date",
+        "gap": 0,
+        "max_delay": 0,
+        "forecast_horizon": 1,
+    }
+    automl = AutoMLSearch(
+        X_train=X,
+        y_train=y,
+        problem_type="time series binary",
+        max_iterations=3,
+        data_splitter=TimeSeriesSplit(n_splits=3),
```

Comment:
TimeSeriesSplit is based on KFold splitter, which means we cannot do 1 split for this scenario. For time series problems, we currently only have support for CV folds

Reply:
I just wanted to verify what this meant, we can still pass in just a `TrainingValidationSplit` for time series can't we?

<img width="1448" alt="Screen Shot 2021-12-22 at 4 41 01 PM" src="https://user-images.githubusercontent.com/29577911/147117869-7feeeed2-8b8f-4dc5-88d6-7f1037c768ee.png">

Reply:
Yep! I just wanted to check whether or not the `TimeSeriesSplit` would be able to do a single TV split. Coverage for the `TrainingValidationSplit` is already ensured through the previous test!